config/zones/TR.yaml

```diff
@@ -120,6 +124,12 @@ emissionFactors:
       datetime: '2021-01-01'
       source: INCER ACV
       value: 26.15
+    geothermal:
+      datetime: '2020-01-01'
+      source: Akın, Serhat, Yasemin Örücü, and Thrainn Frıdrıksson. "Characterizing the declining CO2 emissions from \n
+      Turkish geothermal power plants." (2020)
```

Comment:
This should follow MLA citation style as and should not contain a line break (`\n`). If needed, you can wrap the whole string in single quotes (`''`).